README.md

```diff
@@ -0,0 +1,54 @@
+PR Watcher Notifier
+===================
+
+A flask web application that handles webhook notifications for pull request changes and sends out notifications
+when there are changes to files matched by the configured patterns for each repository
+
+
+Pre-requisites
+==============
+
+Python 3.6 or newer
+
+
+GitHub Webhook setup
+====================
+
+* Use a tool like `ngrok` to expose the local development server on the internet.
+* Create a webhook on the repository to be watched by following [these instructions](https://developer.github.com/webhooks/creating/).
+This requires administrator-level access to the repository.
+* Use the `<ngrok URL>/pull-requests` as the payload URL and set the `Content Type` to `application/json`.
+* Remember to set a secret and to select only the 'Pull request' event.
+* Create a GitHub web token.
+
+
+Development Setup
+=================
+
+* Clone this repository and navigate into the cloned directory.
+* Create a virtualenv environment and install the requirements.
+
+```bash
+$ python3 -m venv venv
+$ source venv/bin/activate
+$ pip install -r requirements.txt -r requirements-dev.txt
+```
+
+* Verify that the unit tests run without errors and pass.
+
+```bash
+$ pytest
```

Comment:
Could you make this `pytest --cov=pr_watcher_notifier` instead?  The current coverage (90%) is sufficient, but it's useful to see.